README.md

```diff
@@ -104,16 +116,17 @@ import (
 )
 
 func main() {
-	accountSid := os.Getenv("TWILIO_ACCOUNT_SID")
+	username := os.Getenv("TWILIO_ACCOUNT_SID")
 	authToken := os.Getenv("TWILIO_AUTH_TOKEN")
 	phoneNumber := "AVAILABLE_TWILIO_PHONE_NUMBER"
+	accountSid := os.Getenv("TWILIO_SUBACCOUNT_SID")
```

Comment:
Should keep the naming similar to what we do in the docs: accountSid, authToken, and subaccountSid.

https://www.twilio.com/docs/iam/api/subaccounts?code-sample=code-send-a-message-from-a-subaccount&code-language=Node.js&code-sdk-version=3.x